views/default/output/tags.php

```diff
@@ -15,11 +15,11 @@
 	unset($vars['entity']);
 }
 
-if (empty($vars['tags']) && (!empty($vars['value']) || $vars['value'] === 0 || $vars['value'] === '0')) {
+if (empty($vars['tags']) && (isset($vars['value']) && $vars['value'] !== false)) {
```

Comment:
Neither replacement in this file looks functionally equivalent. I'd be in favor of splitting this into nested if statements if it would make it more readable.